CIP-xxxx/README.md

```diff
@@ -0,0 +1,145 @@
+---
+CIP: xxxx
+Title: CBOR Tag definition for CIP-0003 ED25519-BIP32 Keys
+Category: MetaData
+Status: Proposed
+Authors:
+    - Steven Johnson<steven.johnson@iohk.io>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/753
+Created: 2024-1-19
+License: CC-BY-4.0
+---
+
+<!-- cspell: words secp -->
+
+<!-- markdownlint-disable MD025-->
+# [CBOR] Tag definition for [CIP-0003] [ED25519-BIP32] Keys
+
+## Abstract
+
+[CIP-0003] defines [ED25519-BIP32] Keys, Key derivation and a signature scheme.
+This CIP defines CBOR Tags and formalizes the Canonical encoding of data structures for [CIP-0003].
+The intention is to have these tags registered in the [IANA CBOR Tag Registry].
```

Comment:
```suggestion
The intention is to have these tags registered in the [IANA CBOR Tag Registry].
```

Is this explicitly a motivation of this CIP or a side-effect of the registry definition CIP?

Reply:
I believe this is an explicit motivation

Reply:
Yes its explicit motivation.  You need a published source of the proposed tags to get IANA to register them.

Reply:
@stevenj @Ryun1 @Crypto2099 as I understand it then the matter is resolved (please unresolve if not)